source/use-cases/sensitive-data-encryption.txt

```diff
@@ -465,7 +464,7 @@ full `JSON Schema for the Medco Medical Management System
       :tabid: java-sync
        
       View the `helper code in Java
-      <https://gist.github.com/ccho-mongodb/088176b1bed3b9e54cdc0c2c3c537d1b>`_.
+      <https://raw.githubusercontent.com/mongodb/docs-assets/DOCSP-json-schema-helper-and-json/JSONSchemaCreator.java>`_.
```

Comment:
Line 352, we have not told users that keyID must use the extended json v2 format for a $binary key. I think most users will paste in the base64 key and get confused. 

Strongly recommend providing a full skeleton of the $binary format as you have in the [example schema](https://raw.githubusercontent.com/mongodb/docs-assets/DOCSP-json-schema-helper-and-json/MedcoMDSchema.json)

Reply:
The code snippet we provided on this line contains a helper `buildEncryptMetadata` that constructs the contents of the encryptMetadata field in the JSON Schema for them which allows the reader to paste the base64 format key id in the `<paste_your_key_id_here>` line.

Is the request to add a note that mentions that the helper code takes care of this for you? @rkumar-mongo